keyboards/montex/info.json

```diff
@@ -0,0 +1,45 @@
+{
+    "keyboard_name": "montex numpad",
+    "url": "https://www.idobao.net/products/ldobao-montex-pad-hot-swappable-mechanical-keyboard-kit",
+    "maintainer": "NachoxMacho",
+    "width": 5,
+    "height": 6,
+    "layouts": {
+        "LAYOUT_numpad_6x5": {
+            "layout": [
+                { "label": "0,0", "x": 0, "y": 0},
+                { "label": "0,1", "x": 1, "y": 0},
+                { "label": "0,2", "x": 2, "y": 0},
+                { "label": "0,3", "x": 3, "y": 0},
+                { "label": "0,4", "x": 4, "y": 0},
+
+                { "label": "1,0", "x": 0, "y": 1.25},
+                { "label": "1,1", "x": 1, "y": 1.25},
+                { "label": "1,2", "x": 2, "y": 1.25},
+                { "label": "1,3", "x": 3, "y": 1.25},
+                { "label": "1,4", "x": 4, "y": 1.25},
+
+                { "label": "2,0", "x": 0, "y": 2.25},
+                { "label": "2,1", "x": 1, "y": 2.25},
+                { "label": "2,2", "x": 2, "y": 2.25},
+                { "label": "2,3", "x": 3, "y": 2.25},
+
+                { "label": "3,0", "x": 0, "y": 3.25},
+                { "label": "3,1", "x": 1, "y": 3.25},
+                { "label": "3,2", "x": 2, "y": 3.25},
+                { "label": "3,3", "x": 3, "y": 3.25},
+                { "label": "3,4", "x": 4, "y": 2.25, "h": 2},
```

Comment:
```suggestion
                { "label": "2,4", "x": 0, "y": 3.25},
                { "label": "3,0", "x": 1, "y": 3.25},
                { "label": "3,1", "x": 2, "y": 3.25},
                { "label": "3,2", "x": 3, "y": 3.25},
                { "label": "3,3", "x": 4, "y": 2.25, "h": 2},
```

Reply:
I kept this and changed `montex.h` to
```c
	K20, K21, K22, K23,      \
	K30, K31, K32, K33, K34, \
...
{ K20,   K21,   K22,   K23,   K34 }, \
```
Which I felt better captured what was actually needed.